payment-request/PaymentAddress/attributes-and-toJSON-method-manual.https.html

```diff
@@ -0,0 +1,84 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://www.w3.org/TR/payment-request/#paymentaddress-interface">
+<title>
+  PaymentResponse.prototype.shippingAddress
+</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../payment-response/helpers.js"></script>
+<script>
+const options = { requestShipping: true };
```

Comment:
setup({ explicit_done: true, explicit_timeout: true });

Reply:
It's added by helpers.js in this case.